src/modules/vault/hooks/details/useVaultTransactionsRequest.ts

```diff
@@ -15,7 +15,9 @@ const useVaultTransactionsRequest = (
     filter: {
       orderBy: 'createdAt',
```

Comment:
add this value by enum

Reply:
and on file `useHomeVaultsRequest.ts` we have same problem

Reply:
Added in other requests as well